src/main/java/menion/android/whereyougo/gui/activity/MainActivity.java

```diff
@@ -53,7 +52,6 @@
 import cz.matejcik.openwig.formats.CartridgeFile;
 import locus.api.objects.extra.Location;
 import locus.api.objects.extra.Waypoint;
-
```

Comment:
Please let the space before "import menion.*"

Reply:
Fixed